lib/rspec/matchers/built_in/respond_to.rb

```diff
@@ -1,8 +1,8 @@
 RSpec::Support.require_rspec_support "method_signature_verifier"
 
-# TODO: Refactor this file to be under our class length
-# rubocop:disable ClassLength
 module RSpec
+  # TODO: Refactor this file to be under our class length
+  # rubocop:disable ClassLength
```

Comment:
So did you check to see if this now appears on the matchers module instead? I think we need two blank lines instead (see: https://github.com/lsegal/yard/issues/689)

Reply:
If that does work we might want an explanation of that, e.g.
```suggestion
  # TODO: Refactor this file to be under our class length
  # Note, blank lines are to hide this from yard
  # rubocop:disable ClassLength
  
  
```
As I have a habit of removing whitespace :joy:

Reply:
Arg you are right 😣. I wanted to avoid two blank lines because:
> lib/rspec/matchers/built_in/respond_to.rb:6:1: C: Layout/EmptyLines: Extra blank line detected.

Two proposal:
```diff
--- a/lib/rspec/matchers/built_in/respond_to.rb
+++ b/lib/rspec/matchers/built_in/respond_to.rb
@@ -1,7 +1,7 @@
-RSpec::Support.require_rspec_support "method_signature_verifier"
-
 # TODO: Refactor this file to be under our class length
 # rubocop:disable ClassLength
+RSpec::Support.require_rspec_support "method_signature_verifier"
+
```

or
```diff
--- a/lib/rspec/matchers/built_in/respond_to.rb
+++ b/lib/rspec/matchers/built_in/respond_to.rb
@@ -1,7 +1,8 @@
 RSpec::Support.require_rspec_support "method_signature_verifier"

-# TODO: Refactor this file to be under our class length
-# rubocop:disable ClassLength
+# @comment
+#   TODO: Refactor this file to be under our class length
+#   rubocop:disable ClassLength
```

Reply:
The top one if it removed the comment, also add line about why the comment is where it is though...

Reply:
I choose the second solution then https://github.com/rspec/rspec-expectations/pull/1178/commits/f7a896dd8960bf6953f1182df8d50bb91cf42afe

Reply:
Yeah no. I didn't mean`@comment` I wanted another line in the existing comment... What does `@comment` even do?

Reply:
Sorry. I switch to your proposal. I have to add extra Rubocop disable flag to avoid Rubocop offense.

This is so complicated for two line of comment... 😣

Reply:
For your question about @comment this is related to https://github.com/lsegal/yard/issues/484#issuecomment-442596174